.travis.yml

```diff
@@ -18,8 +18,9 @@ install:
   - hack/amptools make -f Makefile.refactor.make build
   - make fmt check install
   - make TAG=local build-image
-  - amp platform pull --local
-  - amp platform start --local
+  - amp platform pull
+  - amp platform start all --local
+  - sleep 5
```

Comment:
yes, we can use the --sync option as Joseph suggest in a previous issue, the needed code already exist to do so.